### PR TITLE
Dynamic Dot color

### PR DESCRIPTION
**Pull request by Aydoun**
* make the dot color dynamic

### CODE REVIEW

File: src/components/Quadrant/index.tsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export const Quadrant = () => {
   const quadrantNum: number = d3Config.quadrants.findIndex(
     (item: { route: string }) => item.route === quadrantParam,
   );
-
+  const quadrantColor = d3Config.quadrants[quadrantNum].color;
```
Comment (perlo27) [inline src/components/Quadrant/index.tsx]: now we extracting 2 properties from this obj, so make sense to do distructing
``` const { color: quadrantColor, name: quadrantName } = d3Config.quadrants[quadrantNum];```

File: src/components/Quadrant/styled.ts [hunk 2]
```diff
@@ -36,7 +36,7 @@ export const List = styled.ul`
 `;
 
 export const Dot = styled.span`
```
Comment (perlo27) [inline src/components/Quadrant/styled.ts]: could you add a type for color prop please?
```Dot = styled.span<{ color: string }>```

File: src/components/Quadrant/TechLists.tsx [hunk 3]
```diff
@@ -10,6 +10,7 @@ export interface TechnologiesListProps {
   selected: string | null;
   setSelected: (a: string | null) => void;
   quadrant: string;
+  color: string,
```
Comment (perlo27) [inline src/components/Quadrant/TechLists.tsx]: ```;``` instead of ```,```

File: src/components/Quadrant/styled.ts (hunk 2 above)
Comment (Aydoun) [reply src/components/Quadrant/styled.ts]: I didn't even knew this syntax is possible, tnx

File: src/components/Quadrant/index.tsx (hunk 1 above)
Comment (Aydoun) [reply src/components/Quadrant/index.tsx]: i though about it too...

**Pull request merged**